toolbox/rhods.py

```diff
@@ -85,6 +87,8 @@ def test_jupyterlab(idp_name, username_prefix, user_count: int,
           ods_sleep_factor: Optional. Delay to sleep between users. Default 1.0.
           ods_ci_test_case: Optional. ODS-CI test case to execute.
           ods_ci_exclude_tags: Optional. Tags to exclude in the ODS-CI test case.
+          notebook_cpu: Optional. Number of cores to request for the notebook. The default is in odh-jupyterhub-global-profile.
+          notebook_memory: Optional. Amount of memory to request for the notebook. The default is in odh-jupyterhub-global-profile.
```

Comment:
Nop for `The default is in odh-jupyterhub-global-profile.`, the original content of this file is ignored, we always update the value with `1cpu + 4GB memory` if these args are not passed

Reply:
I think I've fixed that. Let me know if it's not the case.

Reply:
Fixed IIRC